lms/djangoapps/homepage/constants.py

```diff
@@ -0,0 +1,18 @@
+CONFIGS = {
```

Comment:
Changed to single qoutes. However, I just can't blindly change the variable name, without actually confirming it myself. Searching for it in the edx-notifications repo would consume extra effort, which is out of scope.

Reply:
In my opinion, we should not change the code in a unit test story, unless you cannot write unit tests without making the change. In that case, AC of story should change and QA should be involved.

Reply:
Ideally, a dict should not be in constants, because it is not a constant (it is mutable). Its content can be changed from anywhere, and the change will persist. If it is required then, one way is to return dict from a helper function, so that the helper function always returns the same dict without alteration.

Reply:
@zahid-arbisoft the reason why a new file was introduced and the CONFIGS moved to that file is that the same CONFIGS are used in the unit test as well. I wanted a central location from where I can import them both in the unit test file and in the original file.

Yes dictionaries are mutable, but I think we are using CONFIGS as a constant here, I mean we are not altering it. How about I rename the constants.py file to maybe configs.py, because re-introducing the helpers.py method and adding a new method just to return the configs seems a bit extra to me.

Reply:
Resolving this conversation keeping in view the discussion in Slack.